client/src/app.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      //
+    }
+  }
+
+  componentDidMount() {
+    //
+  }
+
+  render() {
+    return (
+      <>
```

Comment:
I'm sure this isn't a big deal since it won't survive breaking out into components, but why use <>...</> instead of div tags to turn this all into one element?

Reply:
interesting read on why use react fragements over divs:
https://getstream.io/blog/react-fragments/

but mostly because I was told its "better" to use fragments than nesting it in a div.